recipe/recipe.yaml

```diff
@@ -45,7 +45,10 @@ requirements:
   host:
     #- python >=3.9
     - python
-    - flit-core ==3.9.0
+    # temp test while waiting for https://github.com/aws/aws-cli/pull/9623 upstream
+    - if: match(python, "<3.14")
+      then: flit-core ==3.9.0
+      else: flit-core ==3.10.0
```

Comment:
not sure if I want to get ahead of upstream's pinning here, mostly testing if this would be enough to get a green build on ci.

since this is in host but not run, maybe this being conditionally pinned wouldn't propagate to any downstream users?